openedx/adg/lms/webinars/models.py

```diff
@@ -158,11 +179,64 @@ class WebinarRegistration(TimeStampedModel):
     user = models.ForeignKey(
         User, verbose_name=_('Registered User'), on_delete=models.CASCADE, related_name='webinar_registrations',
     )
-    is_registered = models.BooleanField(verbose_name=_('Registered'), )
+
+    is_registered = models.BooleanField(default=False, verbose_name=_('Registered'),)
+    is_team_member_registration = models.BooleanField(
+        default=False, verbose_name=_('Is Presenter, Co-Host, or Panelist'),
+    )
+
+    starting_soon_mandrill_reminder_id = models.CharField(
+        default='', max_length=255, verbose_name=_('Scheduled Starting Soon Reminder Id On Mandrill'),
+    )
+    week_before_mandrill_reminder_id = models.CharField(
+        default='', max_length=255, verbose_name=_('Scheduled Week Before Reminder Id On Mandrill'),
+    )
+
+    objects = WebinarRegistrationManager()
 
     class Meta:
         app_label = 'webinars'
         unique_together = ('webinar', 'user')
 
     def __str__(self):
-        return f'User {self.user}, webinar {self.webinar}'
+        return f'User {self.user}, Webinar {self.webinar}'
+
+    @classmethod
+    def create_team_registrations(cls, team_members, webinar, is_webinar_team=True, **kwargs):
```

Comment:
Judging by the name of the function, it seems that its purpose is to only create registrations for webinar team. Then why are we passing in the parameter `is_webinar_team`. It is understood, by the method name, and the parameter `team_members` that the value of `is_webinar_team` will always be true. Therefore, I see no need to pass it as a variable.